assembl/static2/js/app/components/administration/discussion/personalizeInterface/load.js

```diff
@@ -0,0 +1,21 @@
+// @flow
+import type { ApolloClient } from 'react-apollo';
+
+import DiscussionPreferences from '../../../../graphql/DiscussionPreferencesQuery.graphql';
+import type { PersonalizeInterfaceValues } from './types.flow';
+
+export const load = async (client: ApolloClient, fetchPolicy: FetchPolicy) => {
+  const { data } = await client.query({
+    query: DiscussionPreferences,
+    fetchPolicy: fetchPolicy
+  });
+  return data;
+};
+
+export function postLoadFormat(data: DiscussionPreferencesQuery): PersonalizeInterfaceValues {
+  const { discussionPreferences } = data;
+  return {
+    title: discussionPreferences && discussionPreferences.tabTitle,
```

Comment:
Don't you have a warning if `title` value is `undefined` ? (I think it should complain that it is a controlled component and thus need a string value). So I guess : `discussionPreferences ? discussionPreferences.tabTitle : ''`

Reply:
the title can't be undefined